theme.config.tsx

```diff
@@ -1,53 +1,68 @@
-import { DocsThemeConfig, useConfig } from 'nextra-theme-docs'
-import { useRouter } from 'next/router'
-
-
+import React from "react";
+import { DocsThemeConfig, useConfig } from "nextra-theme-docs";
+import { useRouter } from "next/router";
 
 const config: DocsThemeConfig = {
   project: {
-    link: 'https://github.com/PipeOpsHQ/pipeops-documentation',
+    link: "https://github.com/PipeOpsHQ/pipeops-documentation",
   },
-  docsRepositoryBase: 'https://github.com/PipeOpsHQ/pipeops-documentation',
+  docsRepositoryBase: "https://github.com/PipeOpsHQ/pipeops-documentation",
   logo: (
     <>
-      <img src="https://res.cloudinary.com/djhh4kkml/image/upload/v1665733809/Pipeops/Pipeops_bcnyeo.png" alt="Logo" width="120" height="120" />
+      <img
+        src="https://res.cloudinary.com/djhh4kkml/image/upload/v1665733809/Pipeops/Pipeops_bcnyeo.png"
+        alt="Logo"
+        width="120"
+        height="120"
+      />
     </>
   ),
 
-
   useNextSeoProps() {
-    const { route } = useRouter()
-    const { frontMatter } = useConfig() 
-    if (route !== '/') {
+    const { route } = useRouter();
+    const { frontMatter } = useConfig();
+    if (route !== "/") {
       return {
-        titleTemplate: '%s',
-        description: 
-           frontMatter.description || 'PipeOps: the No-code Tool for Cloud Deployments & Infrastructure Management'
-      }
+        titleTemplate: "%s – Pipeops",
+        description:
+          frontMatter.description ||
+          "PipeOps: the No-code Tool for Cloud Deployments & Infrastructure Management",
+      };
     }
   },
 
+  head: () => {
+    const { asPath } = useRouter();
+    const { frontMatter } = useConfig();
+
+    return (
+      <React.Fragment>
```

Comment:
I believe you don't need the `<React.Fragment>` here, this is an example from the previous link I sent you 
![image](https://user-images.githubusercontent.com/65312338/215872564-216bf2b5-92b6-406a-9a4d-7f0021635510.png)
Read more here https://nextra.site/docs/docs-theme/theme-configuration#dynamic-tags-based-on-page

Reply:
I think `React.Fragment` and `<></>`. But, I can change it, if it isn't necessary. 

...and I imported React so I can use the `Fragment` 

Kindly let me know what your thoughts are on this.

Reply:
Using react isn't necessary you can change it we are focused on using nextjs fully.

Reply:
Great. I'll do that now